src/backend/access/transam/xlog.c

```diff
@@ -8207,7 +8211,8 @@ StartupXLOG(void)
 	if (fast_promoted)
 		RequestCheckpoint(CHECKPOINT_FORCE);
 
-	*shmCleanupBackends = true;
```

Comment:
Maybe add some words to the description about how we can reach this place with `shmCleanupBackends` equal to `NULL`?
And similarly about the place above.

Reply:
added